src/modules/wallet/utils/signMessage.js

```diff
@@ -9,7 +9,7 @@ import { signMessageByHW } from './hwManager';
  * @return {string} a signed value of the message
  */
 export const signUsingPrivateKey = ({ message, account }) => {
-  const msgBytes = cryptography.digestMessage(message);
+  const msgBytes = cryptography.ed.digestMessage(message);
   const signature = cryptography.ed.signDataWithPrivateKey(msgBytes, Buffer.from(account.summary.privateKey, 'hex'));
   const result = cryptography.ed.printSignedMessage({
     message,
```

Comment:
I think `ed` should have a better alias I used `encryptDecrypt` but something better can be used so that its easy for any one reading this snipplet to understand the scope.

```suggestion
  const { ed: encryptDecrypt } = cryptography;
  
  const msgBytes = encryptDecrypt.digestMessage(message);
  const signature = encryptDecrypt.signDataWithPrivateKey(msgBytes, Buffer.from(account.summary.privateKey, 'hex'));
  const result =encryptDecrypt.printSignedMessage({
    message,
```

Reply:
:) this ed means `Elliptic curve` related functions `Ed25519` https://en.wikipedia.org/wiki/Elliptic-curve_cryptography

cryptography has namespace for encrypt and decrypt already, so no need to change this?

Reply:
so I think it can be this so that there is only one use of `cryptography`
```suggestion
  const { ed: { digestMessage, signDataWithPrivateKey, printSignedMessage} } = cryptography;
 
  const msgBytes = digestMessage(message);
  const signature = signDataWithPrivateKey(msgBytes, Buffer.from(account.summary.privateKey, 'hex'));
  const result = printSignedMessage({
    message,
```